workflow-templates/python-linters.yaml

```
@@ -0,0 +1,80 @@
+name: Run Python linters
+
+on: [push, pull_request]
+
+jobs:
+  python-files:
+    runs-on: ubuntu-latest
+    outputs:
+      filelist: ${{ steps.python-files.outputs.filelist }}
+    steps:
+
+    - id: python-files
+      run:
+        echo "::set-output name=filelist::$(find . -type f -exec awk ' /^#!.*python/{print FILENAME} {nextfile}' {} +)"
+
+  pylint:
+    runs-on: ubuntu-latest
+    needs: [python-files]
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+
+    - uses: actions/cache@v2
+      with:
+        path: ~/pip-cache
+        key: pip-3.6-${{ github.sha }}
+        # allow cache hits from previous runs of the current branch,
+        # parent branch, then upstream branches, in that order
+        restore-keys: |
+          pip-3.6-
+    - name: Install Requirements
+      run: |
+        python -m pip install --upgrade pip
+        pip --cache-dir ~/pip-cache install pylint htcondor flask jinja2 gunicorn
```

```suggestion
        pip --cache-dir ~/pip-cache install pylint
```

These packages were specific to the HTCondor-CE repository